### DISCUSSION TITLE
스터디 - 6주차 ~7주차 스터디 목록

### DISCUSSION THREAD

**Discussion by vi2920va** [vi2920va/FE_interview#68] [General]
> 6주차는 팀원들 개인 사정으로 쉬어가는 기간 입니다. 저희 스터디는 강제로 하길 권유하지 않기 때문에 다음주 7주차 내용을 여기에 
작성해주시면 됩니다.

## 스터디 일정
12/24(목) 20:30 또는 21:00
21:00 ~ 21:15 팀원들 PR 남긴 거 리뷰 시간
21:15 ~ 21:35 모의 면접, 모의면접은 상대방이 공부한 것으로 기준으로 2개 정도 질문하고 피드백 해준다.
21:35 ~ 21:50 모의 면접을 했던 상대방이 아닌 다른 사람에게도 질문하고 싶은 걸 한다.
21:50 ~ 22:00 해당 주차에 회고록 작성, 프론트엔드 질문 정리 질문 했던 댓글 남기기.

## 스터디 모음

송현 
--
- Network - Why React(내용 수정)
- FrontEnd - HTTP VS HTTPS

남은 
-- 
- JavaScript - closure
- React - useEffect vs useLayoutEffect

두현
--
- FrontEnd - google.com 을 입력하면 일어나는 일

동기
--
- JavaScript - 브라우저의 렌더링 과정(영생님 작업하신 거 리팩토링)

수민
--
- Frontend - Webpack HMR(Hot Module Replacement)
- JavaScript - 실행 컨텍스트

영생
--
- TypeScript - type과 interface

**Comment by NAMU1105** [comment]
남은 - 6주차 스터디 주제

JavaScript - closure
React - useEffect vs useLayoutEffect

**Comment by ttumzzi** [comment]
수민
Frontend - Webpack HMR(Hot Module Replacement)
JavaScript - 실행 컨텍스트

**Comment by ki7348** [comment]
동기
JavaScript - 브라우저의 렌더링 과정

**Comment by whdudtod1273** [comment]
영생
TypeScript - type과 interface

**Comment by Doodream** [comment]
FrontEnd - google.com 을 입력하면 일어나는 일

**Comment by vi2920va** [comment]
여러분 회고록 써주세요

**Reply by vi2920va** [reply]
송현 - 2주만에 팀원들 봐서 좋았습니다.

**Reply by Doodream** [reply]
이야기 하면서 모두들 잘 되었으면 좋겠다는 생각을 많이 했습니다.

**Reply by ki7348** [reply]
아직 준비해야 할 것이 많다고 느꼈고 차근차근 1년 준비해 나가야겠다는 생각이 들었습니다.

**Reply by ttumzzi** [reply]
서로 모르는 부분은 알려주고 아는 부분에 대해서는 토론식으로 얘기를 나눌수 있는 것이 좋았습니다. 공부를 더 열심히 해야겠다는 자극을 받았습니다.

**Reply by whdudtod1273** [reply]
당연한 거지만 아직 모르는 게 많다고 느꼈습니다. 부족한 부분을 꾸준히 매일 채워나가야겠다고 생각했습니다.
